packages/documentation/src/pages/getting-started/common-tasks/new-application.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Creating a new application
+tags: react,redux,application,routing
+---
+
+# Creating a new application
+
+Note: Check out the [Adding a new feature](/getting-started/common-tasks/new-feature) doc to make sure you're trying to add the right type of feature to VA.gov.
+
+## Yeoman generator
+
+The easiest way to create a new application is with the Yeoman application generator. You can use the [walkthrough](/platform/tools/generator) to create your own regular or form application. The manual set up process is below, which you can read through to get a better understanding of the different pieces that are needed.
+
+## Manual set up
+
+Here are the steps for manually setting up a new application. We're going to walk through setting up an application called "my new application."
+
+1. Create a new folder for your application in `src/applications/`. In our case, we're using `my-new-application`:
+
+![Folder tree with new application folder added](../../../images/getting-started/new-application-folder.png)
+
+2. Add a manifest.json file in the new folder. This file contains important information about your application:
+
+```json
+{
+  // Name of your application, not Veteran facing
+  appName: "My new application",
+  // The Webpack entry point to the bundle for your application
+  entryFile: "./my-new-application-entry.jsx",
+  // The Webpack bundle name for your app. Must be unique.
+  entryName: "my-new-application",
+  // The root url where your application lives
+  rootUrl: "/my-new-application",
+}
+```
+
+3. Create the file you referenced in the `entryFile` property above:
+
+```jsx
+import '../../platform/polyfills';
+import './sass/my-new-application.scss';
+
+import startApp from '../../platform/startup';
+
+import routes from './routes';
+import reducer from './reducers';
+import manifest from './manifest';
+
+startApp({
+  url: manifest.rootUrl,
+  reducer,
+  routes,
+  entryName: manifest.entryName,
+});
+```
+
+This file uses our `startApp` helper to create a React/Redux app that has client-side routes handled with `react-router`. You can leave out the routes or reducer if you don't need Redux or `react-router`. There's also a reference to a Sass file, which will set up the styles for your application.
```

Comment:
maybe mention that we use `react-router` v3 since it's so different from 4+

Reply:
I'm a little hesitant to put versions in these docs, since we know they'll change over time and it's hard to keep track of where to update. Even react-router will get upgraded at some point. :-)